packages/os/host-containers-tmpfiles.conf

```
@@ -1,3 +1,4 @@
 d /etc/host-containers 0750 root root -
+d /etc/host-containers/info 0640 root root -
```

It is better to create directories with the execute bit, otherwise they don't work properly unless the current user can override DAC:
```suggestion
d /etc/host-containers/info 0750 root root -
```